gcovr/tests/shadow/_build/Makefile

```diff
@@ -0,0 +1,2 @@
+all:
```

Comment:
Remove this file and add the compiler call to the root makefile.

Reply:
done